include/Jit/EEObjectLinkingLayer.h

```diff
@@ -73,7 +75,18 @@ class EEObjectLinkingLayer : public ObjectLinkingLayerBase {
       NotifyLoadedFtor NotifyLoaded = NotifyLoadedFtor(),
       NotifyFinalizedFtor NotifyFinalized = NotifyFinalizedFtor())
       : NotifyLoaded(std::move(NotifyLoaded)),
-        NotifyFinalized(std::move(NotifyFinalized)) {}
+        NotifyFinalized(std::move(NotifyFinalized)), ProcessAllSections(false) {
+  }
+
+  /// @brief Set the 'ProcessAllSections' flag.
+  ///
+  /// If set to true, all sections in each object file will be allocated using
+  /// the memory manager, rather than just the sections required for execution.
+  ///
+  /// This is kludgy, and may be removed in the future.
+  void setProcessAllSections(bool ProcessAllSections) {
```

Comment:
It's not an override, it will never be called.  Our `EEObjectLinkingLayer` is a copy of the stock Orc `ObjectLinkingLayer` but with the relocation processing stripped out.  The stock Orc `ObjectLinkingLayer` picked up a `setProcessAllSections` method so that it can be called by `OrcMCJitReplacement` which has a method of the same name that is an override.  It would be correct to remove this method and the new field from our copy and propagate all uses of it away to the 'false' value they always have, but I was leery to do so since it would introduce unnecessary diffs vs the stock copy which would complicate future merges of this sort (and also because I'm not entirely sure that we won't find a need for something similar when we point to whatever target the stock Orc is using it for).